website/core/Footer.js

```diff
@@ -109,6 +93,13 @@ class Footer extends React.Component {
                 </a>
               </div>
             )}
+            <div className="social">
+              <a
+                href={`https://www.linkedin.com/company/finosfoundation`}
+                className="linkedin-follow-button">
+                FINOS on LinkedIN
```

Comment:
@maoo - `LinkedIN` should be `LinkedIn` 

👍

Reply:
Fixed!